src/java.base/share/classes/java/lang/Module.java

```diff
@@ -597,6 +597,13 @@ public boolean isExported(String pn, Module other) {
      *
      * <p> This method does not check if the given module reads this module. </p>
      *
+     * <p> A package {@code p} opened to module {@code M} means that code in
+     * {@code M} can do deep reflection on all types in the package.
+     * Further, if {@code M} reads this module, it can obtain a
+     * {@link java.lang.invoke.MethodHandles.Lookup Lookup} object that is allowed to
+     * {@link java.lang.invoke.MethodHandles.Lookup#defineClass(byte[]) define classes}
+     * in package {@code p}. </p>
```

Comment:
I wonder if this should be an apiNote rather method description. There is also the no-arg isOpen method and maybe we should add a note there too. What would you think about linking "deep reflection" to AccessibleObject.setAccesssible(boolean) ?

Reply:
> I wonder if this should be an apiNote rather method description. 

I considered this and no clear cut on this.  I don't have a strong opinion on this.   What do you think?

Reply:
I updated with apiNote, which is clearer.